timeless/reservations/views.py

```diff
@@ -15,6 +15,13 @@
 
 BP = Blueprint("reservations", __name__, url_prefix="/reservations")
 
+class ReservationsListView(views.ListView):
+    """ List the reservation """
+    model = Reservation
+    template_name = "reservations/list.html"
+    context_object_list_name = "reservations"
```

Comment:
@ebercamargo actually I  would prefer default name from base class - `objects`. just to unify the name in all `ListView` templates. wdyt?

Reply:
@vryazanov I just followed what was specified in the problem - ReservationView.